scenariolib/uabot.go

```diff
@@ -0,0 +1,116 @@
+package scenariolib
+
+import (
+	"time"
+	"math/rand"
+)
+
+// DEFAULTTIMEBETWEENVISITS The time for the bot to wait between visits, between 0 and X Seconds
+const DEFAULTTIMEBETWEENVISITS int = 120
+
+type Uabot interface {
+	Run() error
+}
+
+type uabot struct {
+	local          bool
+	conf           *Config
+	scenarioURL    string
+	searchToken    string
+	analyticsToken string
+	random         *rand.Rand
+}
+
+func NewUabot(local bool, conf *Config, scenarioUrl string, searchToken string, analyticsToken string, random *rand.Rand) *uabot {
```

Comment:
`func NewUabot(local bool, scenarioUrl string, searchToken string, analyticsToken string, random *rand.Rand) *uabot {}`

Instead. And I would move the read file logic as a non-exposed function inside the bot too.